plugins/statistics.koplugin/main.lua

```diff
@@ -177,14 +178,25 @@ function ReaderStatistics:addToMainMenu(tab_item_table)
                 end
             },
             {
+                text = _("Reading progress"),
+                callback = function()
+                    UIManager:setDirty(nil, "full")
```

Comment:
Without that line:
`
 UIManager:setDirty(nil, "full")`

On my KPW3 "reading progress" looks like that:

![flush](https://cloud.githubusercontent.com/assets/22982594/19976492/36f0912c-a1f0-11e6-9fd8-3b117a06b47f.png)

On KR Emulator looks ok.